### ISSUE TITLE
3.4 SR1 has lower version number than 3.4 RTM

### ISSUE THREAD

**Issue by EP01** [xamarin/Xamarin.Forms#4886]
### Description

The recently released version 3.4.0.33328 has a lower version number than 3.4.0.1008975. It therefore doesn't show as an update in nuget. Was this intentional?

**Comment by JasonBro** [comment]
I came across this problem too. I selected version 3.4.0.33328, installed it and it works fine, but it is downgrade in version numbers and I am prompted to 'Update' to 3.4.0.1008975 in the NuGet Updates section. 

Hardly anyone has installed this version as you only notice the release if you look at the Xamarin.Forms release notes.

This is from [NuGet](https://www.nuget.org/packages/Xamarin.Forms/):-

4.0.0.8055-pre1	5,865	a month ago
**3.4.0.1008975	110,326	2 months ago**
3.4.0.987044-pre2	1,855	2 months ago
3.4.0.925479-pre1	6,118	3 months ago
**3.4.0.33328	0	a day ago**
